src/test/scala/modbat/mbt/ModbatTestHarness.scala

```diff
@@ -20,12 +20,38 @@ import modbat.log.Log
 import modbat.util.CloneableRandom
 
 object ModbatTestHarness {
+  def getNamesofFile(args: Array[String], td: org.scalatest.TestData): (String, String, String) = {
+    // Be careful, this function create folder if doesn't exist
```

Comment:
Write a comment at the beginning of the function, describing its overall purpose, and the meaning of the return values. At the end, you can add
"This function creates a directory if it does not exist."